hbase-server/src/main/java/org/apache/hadoop/hbase/procedure2/store/region/RegionProcedureStore.java

```diff
@@ -112,17 +112,15 @@
 
   static final String LOGCLEANER_PLUGINS = "hbase.procedure.store.region.logcleaner.plugins";
 
-  private static final String DATA_DIR = "data";
-
   private static final String REPLAY_EDITS_DIR = "replay";
```

Comment:
Looking over at the MTTR HFile patch, it names the recovered hfile dir 'recovered:hfiles' instead of 'recovered.edits'. This dir could be recovered.wals?

Reply:
Yes, this will be a good name. Let me change.